libbeat/autodiscover/common/config.go

```diff
@@ -0,0 +1,63 @@
+package common
```

Comment:
I worry a bit that this will conflict with our libbeat common package in some places.